### ISSUE TITLE
PMID:10537193 Multiple amino acid substitutions in lanosterol 14alpha-demethylase contribute to azole resistance in Candida albicans.

### ISSUE THREAD

**Issue by CuzickA** [PHI-base/curation#99]
A chemistry resistance paper to curate. Selected from the list of anti-infective papers. 

Current link https://canto.phi-base.org/curs/6eed98a9c5615b1a

Note: session currently contains dummy entry UniProt for Fg TRI5 as I was testing out the PHI-Canto email templates.

**Comment by CuzickA** [comment]
Hi @martin2urban, this publication was entered into PHI-Canto but not curated yet. You mentioned this morning that you would like to try using PHI-Canto for curating a chemistry paper. Perhaps you would like to try this one. I have reassigned the session to you so you should get an email with the details. We can track any questions on this GitHub issue. If you prefer to try and curate another publications that's also fine and we can open a new GitHub issue.

**Comment by martin2urban** [comment]
Hi @CuzickA, this publication does not report on gene modification, but on natural variants. Perhaps not suitable for curation at this time.

**Comment by CuzickA** [comment]
Hi @martin2urban, thanks for making a note of this. I will look through the other two chemistry sessions that we have curated to remind myself about this. It would be good to have clear instructions about which types of chemistry papers we can curate. For now it may be better for you to choose a paper with gene modifications in it to get started.

Here are the links to the other chemistry sessions
https://github.com/PHI-base/curation/issues/51  https://canto.phi-base.org/curs/92ed9bf55f48e444/ro/
https://github.com/PHI-base/curation/issues/53  https://canto.phi-base.org/curs/b06680e815f64df0/ro/

**Comment by CuzickA** [comment]
In both previously curated sessions #51 and #53 only gene modification expts were curated.

I had a scan through this paper and agree with @martin2urban that it is not currently suitable for curation.
It contains information on
1) Natural sequence data from natural strain variants but no gene modifications within the organism.
2) Interspecies transformation expts Ca ERG11 natural variants into Sc.


Closing ticket for now. I have triaged it in not curatable 'other'
